src/sort_cards.py

```diff
@@ -0,0 +1,29 @@
+"""The sort cards module sorts cards from Ace to King."""
+
+
+def sort_cards(cards):
+    """The module takes in a list of cards, and sorts it from Ace to King."""
+    for idx, val in enumerate(cards):
+        if val == 'A':
+            cards[idx] = '1'
+        elif val == 'T':
+            cards[idx] = '9.5'
+        elif val == 'J':
+            cards[idx] = '9.6'
+        elif val == 'Q':
+            cards[idx] = '9.7'
+        elif val == 'K':
+            cards[idx] = '9.8'
+    cards.sort()
```

Comment:
"Ahh!  Help!  I'm a list coming into a function and someone is mutating me!  Why, oh why, did they not use the sorted function instead?"